src/cluster.c

```diff
@@ -1117,7 +1117,7 @@ void clusterBlacklistAddNode(clusterNode *node) {
 }
 
 /* Return non-zero if the specified node ID exists in the blacklist.
- * You don't need to pass an sds string here, any pointer to 40 bytes
+ * You don't need to pass a sds string here, any pointer to 40 bytes
```

Comment:
I think this should be `an`.

```
Abbreviations that start with the consonants F, H, L, M, N, R, S, and X attract an because they start with vowel sounds. For example: 
```